webclient/javascripts/views/scoreboard.js

```diff
@@ -0,0 +1,69 @@
+/** @jsx React.DOM */
+
+var React = require('react');
+var socket = io.connect('http://localhost:3000');
+
+var Scoreboard = React.createClass({
+  getInitialState: function () {
+    socket.on('point', this.fetchState);
+
+    $.getJSON('/api/games/current', this.fetchState);
+    $.getJSON('/api/players', this.fetchPlayers);
+
+    return {
+      score_left: 0
+    , score_right: 0
+    , left: {}
+    , right: {}
+    };
+  },
+  fetchState: function (data) {
+    if (!this.players) {
+      return;
+    }
+
+    data.left = _.findWhere(this.players, {id: data.left});
+    data.right = _.findWhere(this.players, {id: data.right});
+
+    this.setState(data);
+  },
+  fetchPlayers: function (data) {
+    this.players = data;
+  },
+  render: function () {
+    return (
+      <div className="full-expanded scoreboard-layout">
+        <div className="scoreboard scoreboard--flex absolute-positioned">
```

Comment:
We can split the scoreboard in to pieces. The Scoreboard and two ScoreboardSides